onnxruntime/core/mlas/lib/mlasi.h

```diff
@@ -928,6 +952,8 @@ MlasCastToInt32x4(MLAS_FLOAT32X4 Vector)
     return _mm_cvttps_epi32(Vector);
 #elif defined(MLAS_VSX_INTRINSICS)
     return vec_cts(Vector, 0);
+#elif defined(MLAS_WASMSIMD_INTRINSICS)
+    return (MLAS_INT32X4) __builtin_convertvector((__f32x4)Vector, __u32x4);
```

Comment:
nit: can you remove the space before __builtin?

Reply:
sure.

Reply:
is there a standard setup of code formatter for the code base?